runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -416,12 +422,7 @@ private void updateList(ArrayList<String> list, String input)
 			Matcher m = ROTATION_REGEX.matcher(input);
 			while (m.find())
 			{
-				String rotation = m.group(1).toLowerCase();
-
-				if (!list.contains(rotation))
-				{
-					list.add(rotation);
-				}
+					list.add(m.group(1).toLowerCase().replaceAll("\\s|\"", ""));
```

Comment:
De-indent by one tab.
```suggestion
				list.add(m.group(1).toLowerCase().replaceAll("\\s|\"", ""));
```